gamestonk_terminal/fundamental_analysis/fa_controller.py

```diff
@@ -203,14 +198,60 @@ def call_cal(self, other_args: List[str]):
         """Process cal command"""
         yahoo_finance_view.calendar_earnings(other_args, self.ticker)
 
-    def call_av(self, _):
-        """Process av command"""
-        ret = av_controller.menu(self.ticker, self.start, self.interval)
+    def call_overview(self, other_args: List[str]):
+        """Process overview command"""
+        av_view.overview(other_args, self.ticker)
 
-        if ret is False:
-            self.print_help()
-        else:
-            return True
+    def call_key(self, other_args: List[str]):
+        """Process overview command"""
+        av_view.key(other_args, self.ticker)
+
+    def call_income(self, other_args: List[str]):
+        """Process income command"""
+        av_view.income_statement(other_args, self.ticker)
+
+    def call_balance(self, other_args: List[str]):
+        """Process balance command"""
+        av_view.balance_sheet(other_args, self.ticker)
+
+    def call_cash(self, other_args: List[str]):
+        """Process cash command"""
+        av_view.cash_flow(other_args, self.ticker)
+
+    def call_earnings(self, other_args: List[str]):
```

Comment:
not your fault, but earnings has an extra new line. Could you get rid of it since you're at it :p